src/components/PrivateLayout/PrivateLayout.tsx

```diff
@@ -223,27 +242,23 @@ const PrivateLayout = ({ children }: any) => {
                     Статистика округ
                   </Menu.Item>
                 </SubMenu>
-                {/* <SubMenu key="sub2.3" title="Осередки">
-                  <Menu.Item onClick={() => { handleClickAway(); }} key="12">Осередки та адміни</Menu.Item>
-                  <Menu.Item onClick={() => { handleClickAway(); }} key="13">Порівняти осередки</Menu.Item>
-                </SubMenu> */}
               </SubMenu>
-            ) : 
-            ((clubAdm || clubAdmDeputy) ? (
-              <Menu.Item icon={<FileTextOutlined />} onClick={() => { handleClickAway(); history.push(`/annualreport/table/city`); }} key="16">
-                Річні звіти
-              </Menu.Item>
-            ) : (<> </>))
+            ) :
+              (userAnnualReportAccess?.ViewAnnualReportsTable == true ? (
+                <Menu.Item icon={<FileTextOutlined />} onClick={() => { handleClickAway(); history.push(`/annualreport/table/city`); }} key="16">
+                  Річні звіти
+                </Menu.Item>
+              ) : (<> </>))
             }
             {(canEdit === true || canSee === true || canAccess === true || regionAdm === true || cityAdm === true || clubAdm === true) ? (
```

Comment:
I guess it's not your code, but for what reason there are that `=== true`?

Reply:
Thanks, changed it